eZ/Publish/Core/Persistence/Legacy/Tests/TestCase.php

```diff
@@ -114,21 +129,22 @@ protected function getDsn()
     protected function setUp(): void
     {
         try {
-            $handler = $this->getDatabaseHandler();
```

Comment:
Well... yes, we have integration tests done as a part of unit tests. This needs to be rectified, because we cover here SQLite only, but it's a separate scope.